src/core/gui/FloatingToolbox.cpp

```diff
@@ -105,14 +112,15 @@ void FloatingToolbox::showForConfiguration() {
 
 
 void FloatingToolbox::show() {
+    // TODO really??
     gtk_widget_hide(this->floatingToolbox);  // force showing in new position
-    gtk_widget_show_all(this->floatingToolbox);
+    gtk_widget_show(this->floatingToolbox);
 
     if (this->floatingToolboxState != configuration) {
         gtk_widget_hide(this->mainWindow->get("labelFloatingToolbox"));
     }
 
-    if (this->floatingToolboxState == configuration || countWidgets() > 0) {
+    if (this->floatingToolboxState == configuration || hasWidgets() > 0) {
```

Comment:
All usages of hasWidgets should be reviewed (now a bool). So don't compare them with an int.